test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -907,9 +908,9 @@ arbitraryActionWithVars _ findVars _st = QC.oneof $ concat [
     withoutVars :: [Gen (Any (LockstepAction (ModelState h)))]
     withoutVars = [
           Some . New @k @v @blob <$> QC.arbitrary
-        , fmap Some $ Open @k @v @blob <$> genSnapshotName
-        , fmap Some $ DeleteSnapshot <$> genSnapshotName
-        , pure $ Some ListSnapshots
+        -- , fmap Some $ Open @k @v @blob <$> genSnapshotName
+        -- , fmap Some $ DeleteSnapshot <$> genSnapshotName
+        -- , pure $ Some ListSnapshots
```

Comment:
These were commented out because I've removed the `expectFailure` in the top-level test, so I don't want to generate actions that have an `undefined` implementation. I'll add an explanation